src/components/SubjectCard/SubjectCard.js

```diff
@@ -34,30 +35,38 @@ const SubjectCard = ({
   fileName,
   postDate,
   checked
-}) => (
-  <Wrapper>
-    <Image>
-      {findIcon(type)}
-    </Image>
-    <Content>
-      <TaskName>{taskName}</TaskName>
-      <Description>{description}</Description>
-      {!!fileName && <FileName>Nome do arquivo:<span>{fileName}</span></FileName>}
-    </Content>
-    <RightContent>
-      <Date>{postDate}</Date>
-      <CheckBox type='checkbox' checked={checked} />
-    </RightContent>
-  </Wrapper>
-)
+}) => {
+  const [isChecked, setCheck] = useState(checked)
+
+  const handleCheck = () => {
+    setCheck(!isChecked)
+  }
+
+  return (
+    <Wrapper>
+      <Image>
+        {findIcon(type)}
+      </Image>
+      <Content>
+        <TaskName>{taskName}</TaskName>
+        <Description>{description}</Description>
+        {!!fileName && <FileName>Nome do arquivo:<span>{fileName}</span></FileName>}
```

Comment:
Aqui transformar o fileName em boolean tem algum propósito especial ? Se ele for falsy/truthy vai ser false/true de qualquer jeito.

Reply:
Isso tava aí já mas acho que não faz diferença